linode/manager
# fix: Typescript types of some Cypress helpers

## Description 📝
- Some Typescript types for some of our Cypress helpers have incorrect typings
  - **Note** We don't typecheck our cypress tests (maybe we should in the future)
 
## Major Changes 🔄
- Change `Map`s to `Record`s where applicable 
- Removes jsdoc-like notations because I don't think they are giving us any value because we have Typescript

## Preview 📷

### Before
![Screenshot 2023-09-14 at 1 43 25 PM](https://github.com/linode/manager/assets/115251059/c6756907-8b88-435a-9536-e5a778efb429)

### After
![Screenshot 2023-09-14 at 1 43 10 PM](https://github.com/linode/manager/assets/115251059/5e934bee-e6f1-4673-ae32-dd1ff53ff0ad)

## How to test 🧪
- Check types of `visitWithLogin` in your IDE
- Make sure types are what we actually desire (is `Record` the best fit here?)
- Verify e2e test still run as expected (they are hanging in Jenkins at the time of writing so you must test locally)
- The overall risk of this change is very low because it strictly touches types and comments
